backend/src/index.ts

```diff
@@ -27,49 +28,8 @@ app.use(
   })
 );
 
-// Define the MongoDB connection URL
-const dbUrl = process.env.DATABASE_URL;
-
-if (!dbUrl) {
-  console.error('MongoDB connection URL is missing in the .env file.');
-  process.exit(1);
-}
-
-mongoose.connect(dbUrl);
-
-const db = mongoose.connection;
-
-db.on('error', (error) => {
-  console.error('MongoDB connection error:', error);
-});
-
-db.once('open', () => {
-  console.log('Connected to MongoDB');
-});
-
-// TODO: This should not be here. This should be imported from models/user.ts instead
-// Define user model. Email address is the primary, username should be unique too.
-interface IUser {
-  username: string;
-  email: string;
-  password: string;
-}
-
-// TODO: This should not be here. This should be imported from models/user.ts instead
-const userSchema: Schema<IUser & Document> = new Schema({
-  username: { type: String, unique: true },
-  email: { type: String, unique: true },
-  password: String
-});
-
-// TODO: This should not be here. This should be imported from models/user.ts instead
-const User: Model<IUser & Document> = mongoose.model<IUser & Document>(
-  'User',
-  userSchema
-);
-
-// TODO: This should not be here. This should be exported from models/user.ts instead
-export default User;
+// Connect MongoDB
+Db
```

Comment:
Could we replace this with a function imported from `databaseConnection` instead? Maybe something like `connectToDatabase` or similar.

This would make the code more readable and intuitive. Just having `Db` right here looks really odd.

Reply:
Okay